django/contrib/admin/utils.py

```diff
@@ -426,7 +426,7 @@ def help_text_for_field(name, model):
     return help_text
 
 
-def display_for_field(value, field, empty_value_display):
+def display_for_field(value, field, empty_value_display, is_link=False):
```

Comment:
I think a better name might be `avoid_link` or `prevent_link`.

Reply:
Sure, I will push an update :+1: